python/pylibcugraph/setuputils.py

```diff
@@ -245,3 +245,23 @@ def get_repo_cmake_info(names, file_path):
 def _get_repo_path():
```

Comment:
Is there any way you could share the setuputils between cugraph and pylibcugraph? Maybe not, since they're completely separate packages, just asking since it would be nice to remove the duplication.

Reply:
Since they are two separate packages, I think each should have its own `setuputils`. I will talk to Rick to see if we can remove duplicate functions definition and if yes, where they should be define.

Reply:
I can't think of a clean way to share this code without either introducing a shared dependency for this, or updating `sys.path` to allow it to import a module up the source tree that both packages have access to. I'd like to also remove the duplication, so I'll keep thinking of a better way since I'm assuming nobody would prefer my suggestions above over the current copy-paste reuse :)

Reply:
That's fine then. I see the same logic in cuML, and cudf has some of the same functions directly in setup.py. I have previously wondered if in addition to the CMake code in rapids-cmake there might not also be a use case for a small standalone Python module (not even a package) of setup.py helpers that could be shared across RAPIDS libs. This is definitely making me reconsider that possibility, but it's out of scope for now. I'm good to leave this as is for the moment.